### PR TITLE
Improve a bit window loading on init

### PR DESCRIPTION
**Pull request by astorije**
Follow-up of https://github.com/thelounge/lounge/pull/1889#discussion_r158595946.

### CODE REVIEW

**Comment by xPaw** [comment]
Have you looked through the code that `target` is not referenced anywhere?

File: client/js/socket-events/init.js [hunk 1]
```diff
@@ -64,8 +64,7 @@ function openCorrectChannel(clientActive, serverActive) {
 
 	// Open window provided in location.hash
 	if (target.length === 0 && window.location.hash) {
-		target = $("#footer, #sidebar, #help")
-			.find(`[data-target="${escape(window.location.hash)}"]`);
+		target = $("body").find(`[data-target="${escape(window.location.hash)}"]`);
```
Comment (xPaw) [inline client/js/socket-events/init.js]: Either use `document.body` or you can do `$(`[data-target="...` directly.

**Comment by astorije** [comment]
> Have you looked through the code that target is not referenced anywhere?

I did before, but not carefully enough. Will do it better today.

File: client/js/socket-events/init.js (hunk 1 above)
Comment (astorije) [reply client/js/socket-events/init.js]: You're right, will do the latter. I'll also add a `.first()` too, just in case.

Comment (astorije) [reply client/js/socket-events/init.js]: Done.

**Comment by astorije** [comment]
Actually @xPaw, I think it might be used here: https://github.com/thelounge/lounge/blob/astorije/improve-init/client/js/socket-events/quit.js#L14

What do you think? There are a looot of uses of `target` in our client code, it's hard to see the big picture.

**Comment by xPaw** [comment]
That code works on the sidebar doesnt it? Should be fine.

**Comment by astorije** [comment]
Doesn't it remove the current window when quitting?

**Comment by astorije** [comment]
@xPaw, I confirm that this breaks disconnecting from the lobby context menu (right click context menu still works though) :/
So I guess I'll have to leave it there (the target part).

**Pull request merged**